basics_of_logging_in_sst-core/Example03/ExampleComponent.h

```diff
@@ -0,0 +1,67 @@
+#ifndef MY_COMPONENT_H
+#define MY_COMPONENT_H
+
+#include <sst/core/component.h>
+#include <sst/core/output.h>
+#include <sst/core/params.h>
+#include <cstdarg>
+#include "Logger.h"
+
+using namespace LoggerExample;
+
+namespace Example03
+{
+    // This is a very simple component.  It only registers a clock and 
+    // prints log messages as the clock handler is called.
+    //
+    // Remember, all components inherit from SST::Component
+    //
+    class ExampleComponent : public SST::Component
+    {
+        public:
+          
+            // Constructor/Destructor
+            //
+            ExampleComponent(SST::ComponentId_t id, SST::Params &params);
+            ~ExampleComponent() {}
+
+            // Standard SST::Component functions.  These all need to
+            // be implemented in the component, even if they are empty.
+            //
+            void setup(void);
+            void finish(void);
+
+            // Clock handler.  This is the method called from the clock event.
+            //
+            bool clockTick(SST::Cycle_t cycle);
+
+            // Shared documentation macros. 
+            //
+            SST_ELI_DOCUMENT_PARAMS(
+                { "clock", "Component clock rate", "1GHz" },
+                { "clockTicks", "Number of times the handler is called before ending.", "10" }
+            )
+            
+            SST_ELI_REGISTER_COMPONENT(
+                ExampleComponent,                       // Class name
+                "example",                              // Library name (the *.so)
+                "ExampleComponent",                     // Name used to reference the component.  This can be
+                                                        // whatever you want it to be and will be referenced
+                                                        // in the python configuration file.
+                SST_ELI_ELEMENT_VERSION( 1, 0, 0 ),     // Version number
+                "Clock element example",                // Description                 
+                COMPONENT_CATEGORY_UNCATEGORIZED        // Component category
+            )
+
+        private:
+            // Member variables for this example.
+            //
+            Logger logger;              // For displaying log messages.
```

Comment:
LoggerExample::Logger instead of using namespace LoggerExample above.  This is the only place in the file the namespace would need to be added.